samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/model/cat.dart

```diff
@@ -61,6 +56,11 @@ class _$CatSerializer implements PrimitiveSerializer<Cat> {
         specifiedType: const FullType(bool),
       );
     }
+    yield r'className';
```

Comment:
can you please merge the most recent master and make sure there are only changes to the typescript-axios generator?